lib/game_stats.rb

```diff
@@ -0,0 +1,58 @@
+class GameStats
+  attr_reader :games_collection
+
+  def initialize(games_collection)
+    @games_collection = games_collection
+  end
+
+  def total_score
+    @games_collection.games.map do |game|
+      game.away_goals + game.home_goals
+    end
+  end
```

Comment:
We decided to make the total_score method as a way to compile every total score for every game in our truncated games csv file. This allows us to simply call .min and .max on our total_score method to create our highest_total_score and lowest_total_score methods.